opm/simulators/utils/ParallelRestart.cpp

```diff
@@ -282,6 +283,13 @@ std::size_t packSize(const RestartValue& data, Dune::MPIHelper::MPICommunicator
         +  packSize(data.extra, comm);
 }
 
+std::size_t packSize(const std::chrono::system_clock::time_point&, Dune::MPIHelper::MPICommunicator comm)
+{
+    std::time_t tp;
```

Comment:
Assuming that the size of a time_t does not depend on its contents, I'd say yes: can be merged.